### PR TITLE
Crowdstrike Falcon bug fix

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue
](https://jira-hq.paloaltonetworks.local/browse/XSUP-14394)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeIntel/Integrations/CrowdStrikeFalconIntel_v2/CrowdStrikeFalconIntel_v2.yml [hunk 1]
```diff
@@ -26,6 +26,20 @@ configuration:
   - low
   - medium
   - high
+- additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: C - Fairly reliable
```
Comment (dansterenson) [inline Packs/CrowdStrikeIntel/Integrations/CrowdStrikeFalconIntel_v2/CrowdStrikeFalconIntel_v2.yml:30]: Was not sure what default value to use

File: Packs/CrowdStrikeIntel/Integrations/CrowdStrikeFalconIntel_v2/CrowdStrikeFalconIntel_v2.py [hunk 2]
```diff
@@ -547,12 +564,15 @@ def cs_reports_command(client: Client, args: Dict[str, str]) -> CommandResults:
 
 def main():
     params: Dict[str, str] = demisto.params()
+    reliability = params.get('integrationReliability', 'C - Fairly reliable')
```
Comment (dansterenson) [inline Packs/CrowdStrikeIntel/Integrations/CrowdStrikeFalconIntel_v2/CrowdStrikeFalconIntel_v2.py:567]: validation failed and required to add a DBot score reliability support.

**Comment by dansterenson** [comment]
@yaakovi what should I do with this error? 
`[IN116] - You've added required, the field is 'integrationReliability`

File: Packs/CrowdStrikeIntel/Integrations/CrowdStrikeFalconIntel_v2/CrowdStrikeFalconIntel_v2.yml (hunk 1 above)
Comment (yaakovi) [reply Packs/CrowdStrikeIntel/Integrations/CrowdStrikeFalconIntel_v2/CrowdStrikeFalconIntel_v2.yml:30]: You can ask Dror or Danil

Comment (dansterenson) [reply Packs/CrowdStrikeIntel/Integrations/CrowdStrikeFalconIntel_v2/CrowdStrikeFalconIntel_v2.yml:30]: Spoke to Dror, we agreed to put `defaultvalue: C - Fairly reliable`

**Comment by yaakovi** [comment]
validation failed on new mandatory field

**Pull request merged**